source/_patterns/01-molecules/components/investor-logos.mustache

```diff
@@ -1,27 +1,24 @@
+<ol class="investor-logos">
+  {{#logos}}
+    <li class="investor-logos__item">
 
-<section class="investor-logos">
+      <div class="investor-logos__container">
+        <picture class="investor-logos__picture">
```

Comment:
Might consider using `sizes` within here, as each footer image is a known quantity.

Reply:
The `yaml` states this is a `picture` object, but the markup is placed directly in here rather than included via a partial. Might be a bit confusing?

Reply:
We use this technique elsewhere when classes are required.

Reply:
Ok, will keep this for consistency.